types/i18next-express-middleware/index.d.ts

```diff
@@ -2,6 +2,7 @@
 // Project: http://i18next.com/
 // Definitions by: Cyril Schumacher <https://github.com/cyrilschumacher>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.3
```

Comment:
If i18next-express-middleware uses i18next v2 definitions then specifying TypeScript version 2.3 may be omitted.